app/Kconfig

```diff
@@ -40,15 +40,19 @@ menuconfig ZMK_BLE
 	select BT_PERIPHERAL
 	select BT_GATT_DIS
 	select BT_GATT_BAS
-	select SETTINGS
-	# select BT_SETTINGS
 
 if ZMK_BLE
 
 config ZMK_BLE_INIT_PRIORITY
 	int "Init Priority"
 	default 50
 
+config SETTINGS
+	default y
+
+config BT_SETTINGS
+	default y
+	
```

Comment:
I did not understand the use of exposing of above two configs clearly and trying to understand.
But enabling the following config results in linker error:
```
diff --git a/app/Kconfig b/app/Kconfig
index b60db05..f12a57d 100644
--- a/app/Kconfig
+++ b/app/Kconfig
@@ -48,7 +48,7 @@ config ZMK_BLE_INIT_PRIORITY
        default 50
 
 config SETTINGS
-       default y
+       default n
 
 config BT_SETTINGS
        default y

```

Also from the zephyr Kconfig, looks like SETTINGS will be enabled if we enable BT_SETTINGS unless we disable it manually as per the above Kconfig

Reply:
Indeed. Removed the extra `SETTINGS` line for now.